pkgs/development/python-modules/firebase-admin/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pythonOlder,
+  aiohttp,
+  cachecontrol,
+  cryptography,
+  google-api-python-client,
+  google-cloud-firestore,
+  google-cloud-storage,
+  hatchling,
+  http-ece,
+  pyjwt,
+  requests,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+
+  pname = "firebase-admin";
+  version = "6.6.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```

3.7 support is deprecated